<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AlbumListPage</class>
 <widget class="QWidget" name="AlbumListPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>939</width>
    <height>619</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="albumListTitleLabel">
       <property name="styleSheet">
        <string notr="true">font: 36pt &quot;Arial&quot;;
font: 36pt &quot;Hei&quot;;
font: 36pt &quot;Heiti SC&quot;;</string>
       </property>
       <property name="text">
        <string>Albums</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>623</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>28</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="font">
      <font>
       <family>.AppleSystemUIFont</family>
      </font>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page_2"/>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>1-50</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>758</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="pagerLabel">
       <property name="text">
        <string>1 2 3 4 5 ... 10</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
